src/infrastructure/database/modules/postgresql/types/postgres-database.types.ts

```
@@ -0,0 +1,149 @@
+}
+
+/**
+ * PostgreSQL connection configuration
+ */
+export interface PostgresConnectionConfig {
+  connectionString: string;
+  maxConnections?: number;
+  idleTimeout?: number;
+  connectTimeout?: number;
+  maxLifetime?: number;
+  ssl?: boolean | postgres.Options<{}>['ssl'];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)